### DISCUSSION TITLE
Reading user data from directus_users

### DISCUSSION THREAD

**Discussion by bhagya1101** [directus/directus#17791] [Archive]
Hi Guys,
I want something like....
I am creating a flow in which trigger is CRON and then in next step I want to  readd all the users data from `directus_users`, but I cannot see anything that is mentioned about `directus_users` in the operations that I have to perform next.

Then my question is how can I get user's data from `directus_users` after performing CRON as a trigger?

Thanks

**Comment by luciandavidescu** [comment]
System collections are not immediately available in the dropdown options, but you can simply do raw edit in 'read data' and add `directus_users` manually

**Reply by bhagya1101** [reply]
thanks!!

**Comment by delivey** [comment]
Seems like that does not properly save the flow. Is anyone else running into this issue?

**Reply by ched-dev** [reply]
I was having the issue on 10.3.0. It seems the "Toggle Raw Editor" doesn't save the value correctly, but if you use the dropdown and "Edit Raw Value" it saves. 

I filed a bug for this: https://github.com/directus/directus/issues/18974
